internal/pkg/plugin/plugin.go

```diff
@@ -27,18 +27,27 @@ type pluginInfo struct {
 
 // SearchPath goes through the files in the user's $PATH and checks if they are plugins
 func SearchPath(cfg *v1.Config) map[string][]string {
-	log.CliLogger.Debugf("Searching `$PATH` for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
+	if runtime.GOOS == "windows" {
+		log.CliLogger.Debugf("Searching `$PATH` and `%%USERPROFILE%%\\.confluent\\plugins` for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
+	}
+	log.CliLogger.Debugf("Searching `$PATH` and `~/.confluent/plugins` for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
```

Comment:
We're inconsistent on how we format environment variables and file names. My initial reaction is not to format them (and only format commands/flags with backticks, according to https://github.com/confluentinc/cli/blob/main/internal/pkg/errors/README.md#general-message-format)

```suggestion
	log.CliLogger.Debugf("Searching $PATH and ~/.confluent/plugins for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
```